apps/extension/src/webview/index.tsx

```
@@ -0,0 +1,1709 @@
+		if (!vscode) return;
+
+		const handleMessage = (event: MessageEvent) => {
+			const message: WebviewMessage = event.data;
+			console.log('📨 Received message from extension:', message);
+
+			// Handle response to a pending request
+			if (message.requestId && pendingRequests.has(message.requestId)) {
+				const { resolve, reject, timeout } = pendingRequests.get(
+					message.requestId
+				)!;
+				clearTimeout(timeout);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)